tests/Integration/inc/ThirdParty/Plugins/Optimization/AMP/excludeScriptFromDelayJs.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\ThirdParty\Plugins\Optimization\AMP;
+
+use Mockery;
+use WP_Rocket\Admin\Options_Data;
+use WP_Rocket\Engine\CDN\Subscriber;
+use WP_Rocket\ThirdParty\Plugins\Optimization\AMP;
+use WPMedia\PHPUnit\Unit\TestCase;
+
+/**
+ * @covers \WP_Rocket\ThirdParty\Plugins\Optimization\AMP::exclude_script_from_delay_js
+ * @group  ThirdParty
+ * @group  WithAmp
+ */
+class Test_ExcludeScriptFromDelayJs extends TestCase
+{
+	/**
+	 * @dataProvider providerTestData
+	 */
```

Comment:
```suggestion
	/**
	 * @dataProvider configTestData
	 */
```

You can use the configTestData method as the data provider, it will automatically match with the corresponding file in the fixtures directory